go.graphics.swing/src/go/graphics/swing/event/swingInterpreter/GOSwingEventConverter.java

```diff
@@ -253,7 +253,7 @@ private String getKeyName(KeyEvent e) {
 				text = "BACK_SPACE";
 				break;
 			default:
-				text = "";
+				text = "" + e.getKeyChar();
```

Comment:
**Example:** I was trying to adjust game speed by hitting `/` and `]`. Resulting key event text contents were the empty string. Surprisingly, same goes for `-` even though it gets handled in that switch.

**Btw:** Modified keys like Shift, Alt etc. don't work at all. This is why I couldn't just press `+` to accelerate the game. We should probably fix that for the sake of i18n, for there are keyboard layouts out there even weirder than german (e.g. french).